.github/workflows/ci-e2e-0.yaml

```
@@ -46,12 +57,45 @@ on:
+        description: Skip tests with Azure provider
+        default: false
+        type: boolean
+      skipCoreDNS:
+        description: Skip tests with CoreDNS provider
+        default: false
+        type: boolean
+      skipEndpoint:
+        description: Skip tests with endpoints provider
+        default: false
+        type: boolean
+    secrets: inherit
```

actionlint (1.7.12)

[error] "secrets" section is scalar node but mapping node is expected

(syntax-check)